scripts/csc_nvidia_smi.nhc

```diff
@@ -79,7 +79,12 @@ function check_nvmsi_number() {
         return 1
     fi
 
-    unkn=$(echo "$NVSMI_HEALTHMON_OUTPUT" | grep -i 'Unknown Error' | wc -l)
+    if $(systemd-detect-virt 2>/dev/null); then
+        unkn=$(echo "$NVSMI_HEALTHMON_OUTPUT" | grep -i 'Unknown Error' | wc -l)
```

Comment:
it's a bit too much copy paste to my liking.
maybe
```
filter="NOMATCH"
 if $(systemd-detect-virt 2>/dev/null); then
    filter="Host Max"
fi
unkn=$(echo "$NVSMI_HEALTHMON_OUTPUT" | grep -i 'Unknown Error' | grep -v "$filter" | wc -l)
```

Reply:
sure